src/main/java/edu/kis/vh/nursery/DefaultCountingOutRyhmer.java

```diff
@@ -0,0 +1,45 @@
+package edu.kis.vh.nursery;
+
+import edu.kis.vh.nursery.stack.IntLinkedList;
+import edu.kis.vh.nursery.stack.IntStackInterface;
+
+public class DefaultCountingOutRyhmer {
+    private IntStackInterface list;
+
+    public DefaultCountingOutRyhmer(IntStackInterface list) {
+        this.list = list;
+    }
+
+    public DefaultCountingOutRyhmer() {
+        list = new IntLinkedList();
+    }
+
+    public int peekaboo() {

```

Comment:
Originally peekaboo had protected visibility!

Reply:
Changed because of creating additional package for stacks implementations

Reply:
peakaboo does not need to be visible for stacks because should not be accessed from stacks (and probably it is not).